recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domu-nfs-no-coproc.cfg

```diff
@@ -0,0 +1,34 @@
+# =====================================================================
+# Example PV Linux guest configuration
+# =====================================================================
+#
+# This is a fairly minimal example of what is required for a
+# Paravirtualised Linux guest. For a more complete guide see xl.cfg(5)
+
+seclabel='system_u:system_r:domU_t'
+
+# Guest name
+name = "DomU"
+
+# Kernel image to boot
+kernel = "/boot/domu/Image"
+
+device_tree = "/xen/domu.dtb"
```

Comment:
Passing the pvr to guest will include correspondent irq, mmio, etc. It is not enough to get something described in dtb for domu. As I understand all that stuff would be included in a separate patch, so this device_tree line should be the part of next patch.